backend/src/openai.ts

```diff
@@ -32,9 +30,13 @@ import {
 	FunctionAskQuestionParams,
 	FunctionSendEmailParams,
 } from './models/openai';
+import chatModelMaxTokens, {
+	countTotalPromptTokens,
+	filterChatHistoryByMaxTokens,
+} from './utils/token';
 
 // tools available to chatGPT
-const chatGptTools: ChatCompletionTool[] = [
+export const chatGptTools: ChatCompletionTool[] = [
```

Comment:
Prefer this export to be at the end of the file rather than inline. Unfortunately I don't _think_ there's a linting rule for this.